src/Featurizers/Structs.h

```diff
@@ -95,6 +95,65 @@ class SingleValueSparseVectorEncoding {
     bool operator!=(SingleValueSparseVectorEncoding const &other) const;
 };
 
+
+/////////////////////////////////////////////////////////////////////////
+
+///  \class        CSREncoding
+///  \brief         Struct that contains information that can be used to
+///                 create a sparse matrix without creating the matrix
+///                 itself. We are using Compressed Sparse Row form
+///                 in this case.
+///
+///                 CSREncoding contains three vectors to express a
+///                 sparse matrix:
+///                 V contains all non-zero element values,
+///                 ColIndex contains column indices of non-zero values,
+///                 RowIndex has one element per row in the matrix and encodes
+///                 the index in V where the given row starts
+///
+
+template <typename T>
+class CSREncoding {

```

Comment:
>CSREncoding [](start = 6, length = 11)

Suppress unintentional copies of this object with the FEATURIZER_MOVE_CONSTRUCTOR_ONLY. While you are at it, can you add TODO comments in SingleValueSparseVectorEncoding and SparseVectorEncoding to add this as well.

Reply:
FEATURIZER_MOVE_CONSTRUCTOR_ONLY added